src/utils/general.ts

```diff
@@ -895,12 +895,12 @@ export async function clearCache() {
   const cacheArray = await api.storage.list();
   let deleted = 0;
 
-  j.$.each(cacheArray, function (index, cache) {
-    if (!utils.syncRegex.test(String(index)) && !/(^tagSettings\/.*)/.test(String(index))) {
-      api.storage.remove(String(index));
+  for (let i = 0; i < cacheArray.length; i++) {
+    if (!utils.syncRegex.test(`${i}`) && !/(^tagSettings\/.*)/.test(`${i}`)) {
+      api.storage.remove(`${i}`);
       deleted++;
     }
-  });
+  }
```

Comment:
`api.storage.list()` apparently has the wrong type defintion it returns an object(`Promise<{ [key: string]: any }>`). Meaning it can not be handled like an array.

Reply:
Always thought template string was faster than constructor or `toString()`. You against it?

> wrong type

I can change it

Reply:
You used object.keys on the wrong thing. it is still an index for loop and not a for of loop, meaning `i` is still only the index and not the object key.

I dont really care, but premature optimization is the root of all evil. ¯\\_(ツ)_/¯

Reply:
Oooh I got you, we need key not an index of it. Why I even thought about index. Guess my IQ is decreasing

Reply:
Done it, might've got too much locked on these `cacheArray` and `index` parameter in jquery loop lmao